qwandaq/src/main/java/life/genny/qwandaq/utils/QwandaUtils.java

```diff
@@ -544,36 +544,66 @@ public Ask generateAskGroupUsingBaseEntity(BaseEntity baseEntity) {
 		entityMessage.setReplace(true);
 
 		// create a child ask for every valid atribute
-		baseEntity.getBaseEntityAttributes().stream()
-				.filter(ea -> defBE.containsEntityAttribute("ATT_" + ea.getAttributeCode()))
+		defBE.getBaseEntityAttributes().stream()
+//		baseEntity.getBaseEntityAttributes().stream()
+//				.filter(ea -> defBE.containsEntityAttribute("ATT_" + ea.getAttributeCode()))
```

Comment:
Remove unused to code to instead of commenting it out.

Reply:
removed comments